src/components/Search/Search.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {TeamsType, UserDataType} from 'types/types';
+
+interface SearchProps<ObjectType> {
+    originalObject: ObjectType[];
+    updateFilteredObject: (updatedObject: ObjectType[]) => void;
+    notifyError: (hasError: boolean) => void;
+    searchProps: string[];
```

Comment:
updated the searchProps to be an array, allowing more than one prop to be searched